strings/ctype-czech.c

```diff
@@ -336,24 +336,24 @@ my_strnxfrm_czech(CHARSET_INFO *cs __attribute__((unused)),
 
 
 /*
-	Neform�ln� popis algoritmu:
-
-	proch�z�me �et�zec zleva doprava
-	konec �et�zce pozn�me podle *p == 0
-	pokud jsme do�li na konec �et�zce p�i pr�chodu 0, nejdeme na
-		za��tek, ale na ulo�enou pozici, proto�e prvn� a druh�
-		pr�chod b�� sou�asn�
-	konec vstupu (pr�chodu) ozna��me na v�stupu hodnotou 1
-
-	na�teme hodnotu z t��d�c� tabulky
-	jde-li o hodnotu ignorovat (0), sko��me na dal�� pr�chod
-	jde-li o hodnotu konec slova (2) a je to pr�chod 0 nebo 1,
-		p�esko��me v�echny dal�� 0 -- 2 a prohod�me
-		pr�chody
-	jde-li o kompozitn� znak (255), otestujeme, zda n�sleduje
-		spr�vn� do dvojice, dohled�me spr�vnou hodnotu
-
-	na konci p�ipoj�me znak 0
+	Neform\xe1ln\xed popis algoritmu:
+
+	proch\xe1z\xedme \xf8et\xeczec zleva doprava
+	konec \xf8et\xeczce pozn\xe1me podle *p == 0
+	pokud jsme do\xb9li na konec \xf8et\xeczce p\xf8i pr\xf9chodu 0, nejdeme na
+		za\xe8\xe1tek, ale na ulo\xbeenou pozici, proto\xbee prvn\xed a druh\xfd
+		pr\xf9chod b\xec\xbe\xed sou\xe8asn\xec
+	konec vstupu (pr\xf9chodu) ozna\xe8\xedme na v\xfdstupu hodnotou 1
+
+	na\xe8teme hodnotu z t\xf8\xedd\xedc\xed tabulky
+	jde-li o hodnotu ignorovat (0), sko\xe8\xedme na dal\xb9\xed pr\xf9chod
+	jde-li o hodnotu konec slova (2) a je to pr\xf9chod 0 nebo 1,
+		p\xf8esko\xe8\xedme v\xb9echny dal\xb9\xed 0 -- 2 a prohod\xedme
+		pr\xf9chody
+	jde-li o kompozitn\xed znak (255), otestujeme, zda n\xe1sleduje
+		spr\xe1vn\xfd do dvojice, dohled\xe1me spr\xe1vnou hodnotu
+
+	na konci p\xf8ipoj\xedme znak 0
```

Comment:
Rather than adding `\x`-escapes which make it unreadable by anyone, we should _translate_ these Czech-language comments into English so that they're readable by everyone. ([Google Translate for this one](https://translate.google.com/?sl=auto&tl=en&text=%0A%09Neform%C3%A1ln%C3%AD%20popis%20algoritmu%3A%0A%09proch%C3%A1z%C3%ADme%20%C3%B8et%C3%ACzec%20zleva%20doprava%0A%09konec%20%C3%B8et%C3%ACzce%20pozn%C3%A1me%20podle%20*p%20%3D%3D%200%0A%09pokud%20jsme%20do%C2%B9li%20na%20konec%20%C3%B8et%C3%ACzce%20p%C3%B8i%20pr%C3%B9chodu%200%2C%20nejdeme%20na%0A%09%09za%C3%A8%C3%A1tek%2C%20ale%20na%20ulo%C2%BEenou%20pozici%2C%20proto%C2%BEe%20prvn%C3%AD%20a%20druh%C3%BD%0A%09%09pr%C3%B9chod%20b%C3%AC%C2%BE%C3%AD%20sou%C3%A8asn%C3%AC%0A%09konec%20vstupu%20(pr%C3%B9chodu)%20ozna%C3%A8%C3%ADme%20na%20v%C3%BDstupu%20hodnotou%201%0A%09na%C3%A8teme%20hodnotu%20z%20t%C3%B8%C3%ADd%C3%ADc%C3%AD%20tabulky%0A%09jde-li%20o%20hodnotu%20ignorovat%20(0)%2C%20sko%C3%A8%C3%ADme%20na%20dal%C2%B9%C3%AD%20pr%C3%B9chod%0A%09jde-li%20o%20hodnotu%20konec%20slova%20(2)%20a%20je%20to%20pr%C3%B9chod%200%20nebo%201%2C%0A%09%09p%C3%B8esko%C3%A8%C3%ADme%20v%C2%B9echny%20dal%C2%B9%C3%AD%200%20--%202%20a%20prohod%C3%ADme%0A%09%09pr%C3%B9chody%0A%09jde-li%20o%20kompozitn%C3%AD%20znak%20(255)%2C%20otestujeme%2C%20zda%20n%C3%A1sleduje%0A%09%09spr%C3%A1vn%C3%BD%20do%20dvojice%2C%20dohled%C3%A1me%20spr%C3%A1vnou%20hodnotu%0A%09na%20konci%20p%C3%B8ipoj%C3%ADme%20znak%200&op=translate))

And then probably delete the Czech-language originals of the comments, unless there are any Czech-speaking developers who strenuously object :sweat_smile: